test-requirements.in

```diff
@@ -1,4 +1,5 @@
 coverage~=4.3.4  # sonarqube has problems parsing 4.4's output https://github.com/nedbat/coveragepy/issues/578
+httpretty
```

Comment:
Nice, I didn't know about this lib, looks really useful 😯

Reply:
Yep, it is very useful and easy to use :-)